### PR TITLE
cancel tiebreak handling - fix for #338

### PR DESCRIPTION
**Pull request by moldover**
Fix for #338.  I also removed logging of 'user' during startup, seems kinda weird and no one has any strong feelings on it.

Just added a fix for #299

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 1]
```diff
@@ -111,9 +111,10 @@ void convertToCdf() {
   // function: tabulate
   // purpose: run tabulation
   // returns: list of winners
-  void tabulate() throws TabulationCancelledException {
+  void tabulate() {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
+    Boolean tabulationSuccess = false;
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: boolean

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 2]
```diff
@@ -143,7 +144,13 @@ void tabulate() throws TabulationCancelledException {
             Logger.log(Level.SEVERE, "Aborting tabulation due to cast vote record errors!");
             break;
           }
-          Set<String> newWinnerSet = runTabulationForConfig(config, castVoteRecords);
+          Set<String> newWinnerSet;
+          try {
+            newWinnerSet = runTabulationForConfig(config, castVoteRecords);
+          } catch (TabulationCancelledException e) {
+              Logger.log(Level.SEVERE, "Tabulation was cancelled by the user!");
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: indentation is off

File: src/main/java/network/brightspots/rcv/TabulatorSession.java (hunk 1 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Why this?  I remember we decided this at some point, but I think we should at least consider using Booleans in RawContestConfig.  It allows us to distinguish between missing (null value, or no key) vs. false values in Jackson.

**Comment by moldover** [comment]
Made edits.  @tarheel what logic are you talking about?  all i did was add a flag which controls a log message.

File: src/main/java/network/brightspots/rcv/TabulatorSession.java (hunk 1 above)
Comment (tarheel) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Sure, I'm open to Booleans (and other object equivalents for primitives) in RawContestConfig if we don't decide to use all Strings.

But in a case like this, there's no reason not to use a primitive; all you want to store is false or true.

**Comment by tarheel** [comment]
> what logic are you talking about? all i did was add a flag which controls a log message.

You changed the level in the stack where the exceptions are caught, and that's not a part of the code that I've thought about before.

**Pull request merged**